Formula/php.rb

```diff
@@ -294,17 +295,20 @@ def plist; <<~EOS
   end
 
   test do
-    assert_match /^Zend OPcache$/, shell_output("#{bin}/php -i"), "Zend OPCache extension not loaded"
+    assert_match /^Zend OPcache$/,
```

Comment:
Generally I'd go for:
```ruby
assert_match /^Zend OPcache$/, shell_output("#{bin}/php -i"),
  "Zend OPCache extension not loaded"
```
rather than dropping it across three lines, but that can be fixed when you pull if you like; doesn't need another CI run to handle.